torch/quantization/fx/_equalize.py

```diff
@@ -86,8 +99,9 @@ def calculate_scaled_minmax(self):
         # Scale the input by the equalization scale located at the same column
         # index
         (min_inputs, max_inputs) = self.get_input_minmax()
-        min_input_scaled = torch.min(torch.mul(min_inputs, self.equalization_scale))
-        max_input_scaled = torch.max(torch.mul(max_inputs, self.equalization_scale))
+        equalization_scale_reshaped = reshape_scale(self.equalization_scale, 0, min_inputs)
```

Comment:
can you add a comment about why we need to do reshapes?

Reply:
Sure, a (hopefully) more in-depth look at the reshaping can be found here: https://fb.quip.com/RMOKAsGdRGKX